src/panoramic/cli/config/analytics.py

```diff
@@ -0,0 +1,31 @@
+import distutils.util
+import os
+from typing import Optional
+
+import requests
+
+from panoramic.cli.supported_version import fetch_analytics_write_key
+
+
+def is_enabled() -> bool:
+    try:
+        enabled = os.environ.get('PANO_ANALYTICS_ENABLED', 'true')
+        return distutils.util.strtobool(enabled)
+    except ValueError:
+        return True
+
+
+def get_write_key() -> Optional[str]:
+    """Get Segment integration source write key."""
+    write_key = os.environ.get("PANO_ANALYTICS_WRITE_KEY", None)
+    if write_key is not None:
+        return write_key
+
+    try:
+        write_key = fetch_analytics_write_key()
```

Comment:
I'm not sure how much we need to worry about this, but it's not a bad idea to have it server-side. It's not really a private key, so I'm not worried about it leaking or anything.

Reply:
yea, I want to store it on remote server just so we can easily change between segment integrations for any reason, like migration, reaching limits, abuse of the key or whatever. If we would hardcode write key to source, users would have to update the version to get a new key and all old versions would be broken.